### PR TITLE
Entities consistency script

### PR DESCRIPTION
**Pull request by fgalan**
Issue #4481

### CODE REVIEW

File: scripts/entities_consistency/test_entities_consistency.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            level=logging.getLevelName('INFO'),
+            format="time=%(asctime)s | lvl=%(levelname)s | msg=%(message)s",
+            handlers=[
+                logging.StreamHandler()
+            ]
+        )
+        logger = logging.getLogger()
+
+        # connect to MongoDB and process validation DB
+        mongo_client = MongoClient('mongodb://localhost:47017')
```
Comment (fgalan) [inline scripts/entities_consistency/test_entities_consistency.py:54]:
```suggestion
        mongo_client = MongoClient('mongodb://localhost:27017')
```

Better to use default port...

File: scripts/managedb/check_entities_consistency.py [hunk 2]
```diff
@@ -1,189 +0,0 @@
...
-#
-# For those usages not covered by this license please contact with
-# iot_support at tid dot es
-
-# Hint: use 'PYTHONIOENCODING=utf8 python check_metadata_id.py' if you are going to redirect the output of this
-# script to a file
-
-# Checks done:
-#
-# 1. heck that there is no duplication in _id fields (for mongo {_id: {x: 1, y: 2}} and {_id: {y:2, x:1}} are different documents)
```
Comment (fgalan) [inline scripts/managedb/check_entities_consistency.py:28]: Rule 15: not swapped subkeys in `_id`

File: scripts/managedb/check_entities_consistency.py [hunk 3]
```diff
@@ -1,189 +0,0 @@
...
-# 2. All attribute in 'attrs' are in 'attrNames' and the other way around
```
Comment (fgalan) [inline scripts/managedb/check_entities_consistency.py:29]: Now in "Rule 14: `mdNames` field consistency"

NTC

File: scripts/managedb/check_location_coherence.py
Comment (fgalan) [inline scripts/managedb/check_location_coherence.py:1]: Now in "Rule 16: `location` field consistency"

NTC

File: scripts/managedb/check_entities_consistency.py (hunk 2 above)
Comment (fgalan) [reply scripts/managedb/check_entities_consistency.py:28]: NTC

File: scripts/entities_consistency/test_entities_consistency.py (hunk 1 above)
Comment (fgalan) [reply scripts/entities_consistency/test_entities_consistency.py]: Fixed in [7845711](https://github.com/telefonicaid/fiware-orion/pull/4490/commits/784571125c47d59e2683cedbed07d25cb817aa12)

File: scripts/entities_consistency/README.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Check [identifiers syntax restrictions](../../doc/manuals/orion-api.md#identifiers-syntax-restrictions) for this case.
+
+### Rule 21: entity type syntax
+
+Check [identifiers syntax restrictions](../../doc/manuals/orion-api.md#identifiers-syntax-restrictions) for this case.
+
+### Rule 22: entity servicePath syntax
+
+https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#entity-service-path
```
Comment (mapedraza) [inline scripts/entities_consistency/README.md:92]:
```suggestion
Check [servicePath documentation](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#entity-service-path)
```

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**